### PR TITLE
Disco inferno P1

### PR DESCRIPTION
**Pull request by Kmc2000**
## About The Pull Request
e

https://streamable.com/z9d8p
## Changelog
:cl: Kmc2000
add: Changed the way ship combat works overall by adding flak, gauss guns (which need people to man them) and reworked PDCs to just be for close quarters defense, rather than a perfect screen against missiles
add: Added FTL console and Tactical console have received a UI change to make them less horrible
tweak: Tweaked the way in which projectiles use their IFFs. Should be more efficient now.
/:cl:

### CODE REVIEW

**Comment by Cenrus** [comment]
I'm gonna vibe check this later

**Comment by Kmc2000** [comment]
Don't worry about the no mapmerge, i'm just lazy

**Comment by Cenrus** [comment]
Clicking the exit verb in the Gauss gun tab should cancel your camera view, we had an MT get stuck in the chair because his camera was locked and had to be cancelled separately.

File: nsv13/code/modules/munitions/ship_weapons/ballistic_weapons/flak.dm [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+		sleep(rand(0, 2))
+	return INITIALIZE_HINT_QDEL
+
+/obj/effect/temp_visual/flak/Initialize()
+	if(prob(50))
+		icon = 'nsv13/goonstation/icons/effects/explosions/96x96.dmi'
+	for(var/obj/X in view(flak_range, src))
+		var/severity = flak_range-get_dist(X, src)
+		if(istype(X, /obj/structure))
+			X.take_damage(severity*10, damage_type = "overmap_light")
```
Comment (Karmic-Skink) [inline nsv13/code/modules/munitions/ship_weapons/ballistic_weapons/flak.dm:158]: Overmap_light is not a damage type.
It is an armour.

Comment (Kmc2000) [reply nsv13/code/modules/munitions/ship_weapons/ballistic_weapons/flak.dm:158]: Wack.

Well it ought to be..but that explains a lot, lol

Comment (Kmc2000) [inline nsv13/code/modules/munitions/ship_weapons/ballistic_weapons/flak.dm:158]:
```suggestion
			X.take_damage(severity*10, damage_type = BRUTE)
```

Comment (Karmic-Skink) [reply nsv13/code/modules/munitions/ship_weapons/ballistic_weapons/flak.dm]: 300% sure I've told you this before.

**Pull request merged**